doc/md_src_files/060_decision-trees.md

```diff
@@ -118,22 +118,20 @@ If nothing else, this means analysts are spending time gathering evidence to mak
 The added details also make it harder for the decision process to accurately manage the risks in question.
 This difficulty arises because more variance and complexity there is in the decision increases the possibility of errors in the decision process itself.
 
+Regarding feature types, all of the features included in SSVC v2 can be considered ordinal data.
+The use of ordinal features is a key assumption behind our use of the parsimony analysis that follows.
+
+By choosing to use only ordinal features in SSVC, 
```

Comment:
This looks like only the first half of a sentence, and the second half should change or replace the following line.

Reply:
Fixed in fddc367907dfdd18d8f37929f557a5da18b23a5f -- I eliminated most of the "By choosing" paragraph and just merged what was left into the paragraph above it.